Nos/Controller/RefreshController.swift

```diff
@@ -1,36 +1,19 @@
 import Foundation
 
-/// Defines a common interface for refreshing.
-@MainActor protocol RefreshController: Observable {
-    /// Whether a refresh should begin or not.
-    var shouldRefresh: Bool { get }
-
-    /// The last time the view was refreshed.
-    var lastRefreshDate: Date? { get }
-
-    /// Update the state of `shouldRefresh` to the given value.
-    func setShouldRefresh(_: Bool)
-
-    /// Updates the last refresh date to the given value.
-    func setLastRefreshDate(_: Date)
-}
-
 /// The default implementation of `RefreshController`.
-@Observable @MainActor class DefaultRefreshController: RefreshController {
+@Observable @MainActor class RefreshController {
+    /// Whether a refresh should begin or not.
```

Comment:
I think I need a better description about this property. The main question I have when seeing this property is: why a controller named RefreshController has a property named shouldRefresh, shouldn't it be always refreshing? Or it is more related to enabling or disabling it?

Reply:
I'll rename it to `startRefresh` and update the comment.

Reply:
Done!